helpers/3p/collection.js

```diff
@@ -19,16 +19,15 @@ var helpers = module.exports;
  *
  * @name .isEmpty
  * @param {Object} `collection`
- * @param {Object} `options`
  * @return {String}
  * @block
  * @api public
  */
 
-helpers.isEmpty = function(collection, options) {
-  if (!options) {
-    options = collection;
-    return options.fn(this);
+helpers.isEmpty = function(collection) {
+  const options = arguments[arguments.length - 1];
+  if (options === collection) {
+    return collection.fn(this);
```

Comment:
It would be more clear what's going on here with something like
```suggestion
  if (arguments.length <= 1) {
    return options.fn(this);
```
or
```suggestion
  if (utils.isOptions(collection)) {
    return options.fn(this);
```